### PR TITLE
Minor grammar fixes

### PR DESCRIPTION
**Pull request by NoahvdAa**
Fixes a couple of things I noticed

### CODE REVIEW

File: source/server/configuration.rst [hunk 1]
```diff
@@ -561,8 +561,8 @@ filter-nbt-data-from-spawn-eggs-and-related
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * **default**: true
 * **description**: Instructs the server to remove certain NBT data from
-  spawn-eggs, falling-blocks, and other often abused items in creative mode.
-* **note**: Some adventure maps may require this be turned off to function
+  spawneggs, falling blocks, and other often-abused items in creative mode.
```
Comment (kennytv) [inline source/server/configuration.rst:564]:
```suggestion
  spawn eggs, falling blocks, and other often abused items in creative mode.
```

File: source/server/configuration.rst [hunk 2]
```diff
@@ -1134,7 +1137,7 @@ anti-xray
 ~~~~~~~~~
 
 .. note::
-   More in depth anti-xray documentation as well as recommended configuration 
+   More in-depth anti-xray documentation as well as a recommended configuration
```
Comment (kennytv) [inline source/server/configuration.rst:1140]: drop the "a" again

**Review by kennytv — CHANGES_REQUESTED** [review]

**Review by kennytv — APPROVED** [review]

File: source/server/configuration.rst [hunk 3]
```diff
@@ -144,7 +144,7 @@ send-full-pos-for-hard-colliding-entities
   client/server disagreement, which may cause glitchy behaviour for players.
   This setting attempts to mitigate this desync by sending precise locations
   for entities involved in collisions. Having this enabled will use more
-  bandwidth, however in the majority of cases, this is a worthy tradeoff.
+  bandwidth, however, in the majority of cases, this is a worthy tradeoff.
```
Comment (Chew) [inline source/server/configuration.rst:147]:
```suggestion
  bandwidth; however, in the majority of cases, this is a worthy tradeoff.
```

**Review by Chew — COMMENTED** [review]
Thank you for the work! Just some more tiny tweaks 😄

**Review by Chew — APPROVED** [review]

**Pull request merged**